package.json

```diff
@@ -196,6 +196,7 @@
     "@storybook/storybook-deployer": "^2.8.6",
     "@testing-library/react": "^10.4.8",
     "@testing-library/react-hooks": "^3.2.1",
+    "@types/react": "^16.9.53",
```

Comment:
I added this as a convenience so that IntelliSense works with types provided via JSdocs and react hooks.